vcloud-blueprint.yaml

```diff
@@ -7,7 +7,7 @@ tosca_definitions_version: cloudify_dsl_1_1
 
```

Comment:
The order of node_templates should be updated to the convention that was established in 3.3m5+. The rules are as follows:

1) First go application nodes
2) Among application nodes, first go front end, then go backend nodes.
3) Then go infrastructure nodes
4) Among, infrastructure nodes, first are compute, then networks, then IPs.
5) Have yet to establish a convention for weird stuff like ssh keys and security groups, but I think it's should be compute, then keys, then groups, but I am open to suggestions.

nodecellar
nodejs:
mongod:
nodejs_host:
mongod_host:
nodejs_security_group:
mongod_security_group:
nodecellar_floatingip:

Since this one is not shared with other nodecellar blueprints, it